src/migrations/20220129113106-metrics-counters-as-bigint.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+exports.up = function (db, cb) {
+    db.runSql(
+        `
+        ALTER TABLE client_metrics_env
+        ALTER COLUMN yes TYPE BIGINT,
+        ALTER COLUMN no TYPE BIGINT;
```

Comment:
Seems fair, that pushes the problem with accuracy over to javascript's number handling, but if your counters start hitting >2^50 in the time we preserve them I think the accuracy won't matter that much

Reply:
I am not sure what we should do then. Sounds like more than 2^50 in an hour for a single toggle in an environment is above what we want to support for now.